docker-compose.yml

```diff
@@ -83,6 +83,7 @@ services:
       &&  mkdir -p /export/gpdeletebeforenamebucket
       &&  mkdir -p /export/gpdeletebeforetimebucket
       &&  mkdir -p /export/storagetoolsbucket
+      &&  mkdir -p /export/restorebucket
```

Comment:
`restorebucket` name is too ambigious, maybe change it to the `walrestorebucket`? Same for the name of the tests and dockerfiles